build/ci.yml

```diff
@@ -4,7 +4,7 @@ trigger:
 
 variables:
   Build.Major: 0
-  Build.Minor: 0
+  Build.Minor: 9
```

Comment:
I wonder how we are going to keep these in-sync.

Reply:
Came here to look what the consensus on this one was - I don't think they need to be in sync, but intended to keep it the same as other QDK repos nonetheless. What do we want to do here?